### ISSUE TITLE
May I ask if the results in the paper are from the eval set or the test set? Please let me know

### ISSUE THREAD

**Issue by marcocst** [TsinghuaC3I/SoRA#6]
May I ask if the results in the paper are from the eval set or the test set? Please let me know

**Comment by telxt** [comment]
Thank you for your interest in our work! The results in the paper are from the test set.

**Comment by marcocst** [comment]
Thank you！

**Comment by sborse3** [comment]
@marcocst were you able to run inference on the test set?

**Comment by telxt** [comment]
@sborse3 @marcocst Sorry for not making it clear earlier. The results in the paper are from the test set. But this test set differs from the test part of the original dataset from Huggingface. We partition the dataset as follows:

For small datasets (n_samples < 10K), we divide validation set to half, use one half as test set and one half as validation set. For larger datasets (n_samples > 10K), we divide training set into 1K as validation and the rest as training set, keeping the original validation set as the test set. You can find the specific implementation in the `get` function within the `SoRA/src/processor.py` file (Lines 87-106).

Please don't hesitate to contact us if you have further questions or need more assistance!
